app/controllers/public_controller.rb

```diff
@@ -3,7 +3,8 @@
 # routes that don't require authentication
 class PublicController < ApplicationController
   def pricing
-    @plans = Rails.cache.fetch('plans', expires_in: 1.hour) do
+    @plans = 
+    Rails.cache.fetch('plans', expires_in: 1.hour) do
```

Comment:
I don't love that this code is identical to https://github.com/jshawl/saas-starter/blob/ma/app/controllers/plans_controller.rb#L6 

but I do like how there is an obvious distinction between the public experience and the authenticated experience. 

What about serving all of this content from `/pricing`? So there's just one page ever that displays pricing info and allows users to checkout?